src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -44,11 +44,11 @@ def handle_slack_error(exception: SlackApiError, endpoint: str, kwargs: dict) ->
     )
     error = exception.response["error"]
 
-    if error in {"channel_not_found", "user_not_in_channel"}:
+    if error in {SlackAPIErrorCode.CHANNEL_NOT_FOUND, SlackAPIErrorCode.USER_NOT_IN_CHANNEL}:
```

Comment:
Is there a reason to use `{}` here? Maybe we could use a `[]` or even better `()` (if we care about non-mutability)